### ISSUE TITLE
[Bug Report][3.3.14] List sub group collapse animation not working in Chrome

### ISSUE THREAD

**Issue by joshas** [vuetifyjs/vuetify#18150]
### Environment
**Vuetify Version:** 3.3.14
**Vue Version:** 3.3.4
**Browsers:** Google Chrome
**OS:** Android

### Steps to reproduce
Click on "Users" to expand sub group. Notice that expand has animation.
Now click on "Users" again to collapse sub group.

### Expected Behavior
Collapse should also have an animation.

### Actual Behavior
Collapse does not have any animation.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrNVbFu2zAQ/RWCixPAslKkk+AECdqhS5cWmSwPjHh22FAkQVJKgsD/3iMl2bJkO0Y7tF4kHu/uvXePphbv9N6YWV0BzejcQ2kk83CbK0LmdVIwy8MrIYVkzt3ktHxNWOV1Tpvwi+D+CcPXV1dNKFbGWimcJ3VSag4y0wYUcEwMLzlts7Z5iUBkYizgLk9EoVWA4iJ50iXklHjhJWDoW1zeztNeGfYadltbXRlSM1mFogcH1vUwMa/Tifyc1D5jhRc189pi+jvy0MaRzV7JPtV+nGCTR6GCuFjYjab7HVDFikJXyieFsIVEQfsFndiWd39zpHxLLu1Z1wsfHMg9L0XfhD8eyamhfDSWnc6Wzv72UaVHtZ52aBWFXCwi6JQEM5b4uCRCERYIjAhmz/CGJWIU73jH52h3YHd4jHI6Iw50OOFw38tzXEYDdZD13/jcEfp3The24ucbfcKmvz8G5xt9ItwFm/saF+HCxsVgcHNXWGGaLHg12nrCYcUq6cl704gzzzJycUlubslFGyMkXNYZWUziTTRZTrt484fBnR3rxeQ7U2wNJSg/mZJJ/5orEUgYCYmuvBQKep1C4U/wXqi168oKvT6QuXuLJg7Av1hAsV0HIyt3BOwHMN6lrcRRTg8GJ7LtVzWr/ZSvIGGXgh+5sBrz3VzGl00wpbWBTilCPrvZL6cVfnbjvHNaODyaW105ffLeuCxNC64wEwFEbWcKfKpMmeIH24vV29317Hr26XPK8RB0sUSyRzeL3UKrZa42iCjK4HtSMjOAbTYidGt8TrFTWH/A4Q7TUosOixISrstIpuXSC8/Alcmj1S94irBJTtvJRJhA+AyoD+Uixqh3ioA12MTiHxEs2HMlDcr6sgZbI2mt1Ru6mdJYhpNvONPlb8vT0tU=)

### Other comments
Works correctly on Firefox on desktop and Android, also on GNOME Web (WebKit). On Chrome Android 116 collapse animation does not work, although sub group expands with animation.

**Comment by KaelWD** [comment]
Works for me, Chrome 116.0.5845.114

**Comment by joshas** [comment]
Yes, I have exactly the same version of Chrome, but somehow on my test device with Android 9 list sub groups are collapsing without any animation. Works correctly on Android emulator, and even on Firefox on Android. Hopefully this is very rare case and users won't stumble upon it. Not really sure if I should close this issue?
